Animations/src/renderer/Renderer.cpp

```diff
@@ -383,7 +385,6 @@ namespace MathAnim
 			g_logger_assert(framebuffer.includeDepthStencil, "Invalid framebuffer. Should include depth and stencil buffers.");
 
 			debugMsgId = 0;
-			glPushDebugGroup(GL_DEBUG_SOURCE_APPLICATION, debugMsgId++, -1, "Main_Framebuffer_Pass");
```

Comment:
I'd like to leave these calls in until the OpenGL compatibility mode gets added https://github.com/ambrosiogabe/MathAnimation/issues/60